apps/web/src/components/forms/marker-create-edit-form.tsx

```
@@ -338,58 +340,78 @@ export default function MarkerForm() {
+                          htmlFor={collection._id}
+                          className="text-sm cursor-pointer flex-1"
+                        >
+                          {collection.title}
+                        </Label>
                       </div>
-                    </Button>
-                  );
-                })}
-            </ScrollArea>
+                    );
+                  })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)